specs/system_config.md

```diff
@@ -89,6 +93,7 @@ The contained log events are filtered and processed as follows:
   - type `1`: `overhead` and `scalar` overwrite, as two packed `uint256` entries.
   - type `2`: `gasLimit` overwrite, as `uint64` payload.
   - type `3`: `unsafeBlockSigner` overwrite, as `address` payload.
+  - type `4`: `l1BasefeeScalar` and `l1BlobBasefeeScalar` overwrite, as two packed `uint32` entries.
```

Comment:
As two packed as in `abi.encodePacked` rather than `abi.encode`?

Reply:
abi.encodePacked is going away apparently, but ideally yes.  OK to use it still?